NDcPP_v3_0.adoc

```diff
@@ -1241,6 +1242,8 @@ _For distributed TOEs that implement a registration channel (as described in FCO
 
 _If the TOE offers ability for a remote authorized IT entities or authorized remote Administrators to connect via an interface secured with SSH, then the ST author must select the option “Ability to manage the trusted public keys database” to account for management of public key authentication. It is acceptable for this management function to be implemented as part of general TOE management functionality or as a standalone management function._
 
+_If the TOE offers ability to modify the public key used to validate the digital update, then the ST author must select the option “Ability to manage the public key or certificate used to validate the digital update”. There is no requirement to implement this as a standalone management function, it is acceptable for this management function to be implemented as part of the trusted update (FPT_TUD_EXT.1) functionality._
```

Comment:
Would this read better if it said, "If the TOE offers the ability to modify..."?

Reply:
@Kirky-J Yes, I agree with your suggested edit.